test/views/organization_membership_view_test.exs

```diff
@@ -0,0 +1,48 @@
+defmodule CodeCorps.OrganizationMembershipViewTest do
+  use CodeCorps.ConnCase, async: true
+
+  import CodeCorps.Factories
+
+  alias CodeCorps.Repo
+
+  # Bring render/3 and render_to_string/3 for testing custom views
+  import Phoenix.View
+
+  test "renders all attributes and relationships properly" do
+    organization = insert(:organization)
+    user = insert(:user)
+    organization_membership = insert(:organization_membership, member: user, organization: organization)
+
+    organization_membership =
```

Comment:
What is the reason for the reassignment? Do the relationships not get loaded through the factory output?